common/helpers.js

```diff
@@ -2,24 +2,47 @@ const url = require('url');
 const config = require('../config.json');
 
 const dataGuidsHostPrefix = 'dg.';
-const dataObjectPathPrefix = '/ga4gh/dos/v1/dataobjects/';
+const dosDataObjectPathPrefix = '/ga4gh/dos/v1/dataobjects/';
+const drsDataObjectPathPrefix = '/ga4gh/drs/v1/objects/';
+const jadeDataRepoHostRegex = /jade(-terra)?.datarepo-(dev|prod).broadinstitute.org/;
```

Comment:
I'd make this regex wilder. If when they add an `aou` or a `staging` repo we don't want to have to deploy a new version. perhaps `/jade.*\.datarepo-.*\.broadinstitute\.org/`?

Reply:
I have widened the regex